### PR TITLE
headbar with tabs

### PR DESCRIPTION
**Pull request by gabrielbursztein**
Headbar with main tab, My books and Log out

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
     devise_for :users
     root 'books#index'
+    resources :requests;
```
Comment (GuilleLeopold) [inline config/routes.rb]: Delete the `;`

**Review by GuilleLeopold — CHANGES_REQUESTED** [review]
I suggest you to add Alexis or Ale to review the CSS.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,128 @@
...
+.wrapper {
+  width: 60%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 15px;
+  border-top: 1px solid rgba(0,0,0,0.2)
+}
+
+.title {
+  font-family: "Times New Roman";
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Use single quotes

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -13,3 +13,128 @@
...
+  font-weight: 900;
+  text-align: center;
+  font-size: 40px;
+}
+
+.book_title {
+  font-family: "Times New Roman";
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Add a variable for this (http://sass-lang.com/guide) after you moved this file to another one with the file type 'scss`

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -13,3 +13,128 @@
...
+  font-weight: 900;
+  font-size: 35px;
+}
+
+.book_author {
+  display: flex;
+  align-content: space-between;
+  font-size: 20px;
+  color: rgba(0,0,0,0.5);
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Move `rgba(0,0,0,0.5)` to a variable

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -13,3 +13,128 @@
...
+  font-size: 12px;
+}
+
+a.request {
+  font-size: 20px;
+  font-family: Arial;
+  float: right;
+}
+
+table {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Don't use html tags, use a class instead

File: app/assets/stylesheets/application.css [hunk 6]
```diff
@@ -13,3 +13,128 @@
...
+  width: 55%;
+  margin-left: auto;
+  margin-right: auto;
+  border-collapse: collapse;
+  font-family: Arial;
+}
+
+tr {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Same as above

File: app/assets/stylesheets/application.css [hunk 7]
```diff
@@ -13,3 +13,128 @@
...
+  line-height: 35px;
+  border-top: 1px solid rgba(0,0,0,0.2);
+}
+
+ul {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Same as above

File: app/assets/stylesheets/application.css [hunk 8]
```diff
@@ -13,3 +13,128 @@
...
+    list-style-type: none;
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Indent with 2 spaces

File: app/assets/stylesheets/application.css [hunk 9]
```diff
@@ -13,3 +13,128 @@
...
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: auto;
+    margin-left: auto;
+    padding-bottom: 10px;
+    width: 90%;
+    border-bottom: 1px solid black;
+}
+
+li {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Don't use html tags, use a class instead

File: app/assets/stylesheets/application.css [hunk 10]
```diff
@@ -13,3 +13,128 @@
...
+  display: inline;
+}
+
+li a {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Don't use html tags, use a class instead. You can use one level of nesting in sass http://sass-lang.com/guide

File: app/assets/stylesheets/application.css [hunk 11]
```diff
@@ -13,3 +13,128 @@
...
+
+li.home_tab {
+  flex:1;
+}
+
+li a:hover:not(.active) {
+  border-bottom: 1px solid black;
+}
+
+li a.main_tab {
```
Comment (MaicolBen) [inline app/assets/stylesheets/application.css]: Use sass nesting http://sass-lang.com/guide

**Review by MaicolBen — CHANGES_REQUESTED** [review]
Looks good! Some things to do:
- Move the css code to another file
- Add `db/development.sqlite3` to the gitignore

Let me know if you've any questions

**Review by GuilleLeopold — APPROVED** [review]

**Review by MaicolBen — APPROVED** [review]

**Pull request merged**